pymilo/pymilo_param.py

```diff
@@ -1,5 +1,51 @@
 # -*- coding: utf-8 -*-
 """Parameters and constants."""
+from sklearn.mixture import BayesianGaussianMixture
```

Comment:
It's not a bug. Rather, it's related to readability.

Instead of importing each class separately, only import the root module.
```python
import sklearn.cluster as cluster
op = cluster.OPTICS(...)
```

Reply:
Sure on my eyes